pywbem/cim_operations.py

```diff
@@ -486,6 +487,32 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     the :attr:`last_raw_request` and :attr:`last_raw_reply` attributes
     of the connection object.
 
+    WBEMConnections can record the results of methods that interact with
+    a WBEM Server using either the :class:`~pywbem.LogOperationRecorder` (uses
+    python logging
+    to create logs for methods and http data) or the
+    :class:`~pywbem.TestClientRecorder`( creates a yaml file of methods and
+    http for tests) which are subclasses of the :class:`~pywbem.Recorder`
+    class.
```

Comment:
1. Not sure we need to mention the superclass here.
2. The superclass is not called "Recorder".